html/create_object.html

```diff
@@ -46,20 +46,19 @@
 		<select name="object_list" id="object_list" size="18" multiple style="width:98%"></select><br>
 		<input type="submit" value="spawn">
 	</form>
-	
+
 	<script language="JavaScript">
 		var old_search = "";
 		var object_list = document.spawner.object_list;
 		var object_paths = null /* object types */;
 		var objects = object_paths == null ? new Array() : object_paths.split(";");
-		
+
 		document.spawner.filter.focus();
-		populateList(objects);

```

Comment:
this is the only change made.

Reply:
this requires explanation. What exactly does this do and why remove it?

Reply:
when opening the game panel, this automatically causes the list of all objects in the game to be displayed.

this happens for a second during which the game freezes and at the end, you don't search within that list but filter the path you want to find via the search engine.

Reply:
In general, this makes the processor cry every time the game panel is opened.